benchexec/tablegenerator/react-table/src/tests/__snapshots__/TaskDefinitionViewer.test.js.snap

```diff
@@ -2,756 +2,611 @@
 
 exports[`Render TaskDefinitionViewer for empty_file.yml 1`] = `
 <pre
-  className="link-overlay-text"
```

Comment:
These changes come from the added json setup declarations in the `package.json`.
These extend the jest snapshot serializer before tests were run. Before this change, the extension of `expect` only happened on import of `tests/utils.js::test_snapshot_of`.

Reply:
All snapshot changes in commit ae5fd37 can be retraced to this change. 
(Apart from the changes in `FilterCard.test.js.snapshot`, as i forgot to update snapshots after fixing sliders and removing the checkmark